New format

mcgregorian1 opened the pull request:
Changed format of processing to use gdal, which is quicker.

mcgregorian1 committed:
New format

Changed format of processing to use gdal, which is quicker.

mcgregorian1 commented on scripts/S5_defineGaps.R:
```diff
@@ -54,6 +54,29 @@ crs(vecRemove) <- "epsg: 32617"
+
+saveGapFiles <- FALSE
+saveGapsPath <- "droneData/processedChange/gapsIndex/fileType/gapsD1_D2_res20.ext"
+
```
sorry, if you're running this there should be this added: (when everything can be merged I'll go back and fix things)

```
saveGapsPath <- ifelse(changeType=="structural", 
                        gsub("gapsIndex", "gapsCanopy", saveGapsPath),
                        saveGapsPath)
```

mcgregorian1 commented:
I can confirm that the gdal-derived structural gaps are the same as the terra-derived structural gaps (tested with looking at 2023-10-12 vs 2023-10-31). This is not the case for the spectral but I'm not sure why; might be a glitch on my part.

kccushman reviewed (approved):
Looks good, thanks for the update!

mcgregorian1 commented:
I am going to close this pull request and instead reformat the script such that the gdal option is the only option. This commit has both still for the purposes of comparison, but we don't need both moving forward and the original method will be accessible via the history of this file.

mcgregorian1 committed:
Updated methods

From S5 - removed terra method for identifying gaps as the gdal method is MUCH faster. Thus, also removed (now)-unnecessary functions in S5_funs along with some file reformatting.

Also from S5, removed some temp file paths that were put in args.R

In args.R, added declamatory statements and reformatted a little.

mcgregorian1 committed:
Merge branch 'main' into ian_gdal
